spec/models/provider_spec.rb

```diff
@@ -46,4 +47,24 @@
     end
     it { is_expected.to allow_values("scitt", "lead_school", "university").for(:provider_type) }
   end
+
+  context "scopes" do
+    describe "#placements" do
+      let!(:onboarded_provider) { create(:provider, placements: true) }
```

Comment:
Not important, but you can do `create(:placements_provider)`

Reply:
I should have remembered that. I wrote the factory. 😛

Reply:
Oh wait. I did that because `create(:placements_provider)` returns a `Placements::Provider` but the scope returns `Provider`. So it needs to be `create(:provider, placements: true)`